ansible/roles/update/templates/52unattended-upgrades.j2

```diff
@@ -2,13 +2,17 @@
 
 // Controls which packages are upgraded
 Unattended-Upgrade::Allowed-Origins {
+{% if 'bigbluebutton' in group_names %}
+        "${distro_id}:${distro_codename}-security";
+{% else %}
         "${distro_id}:${distro_codename}";
         "${distro_id}:${distro_codename}-updates";
         "${distro_id}:${distro_codename}-security";
+{% endif %}
```

Comment:
Why do we need this `if`? Why not just the line `${distro_id}:${distro_codename}-security";`?

The whole point of this role is to install BBB. So we might only say that it needs the security updates.

Reply:
This template is being used by all other hosts also.
In the case of bigbluebutton host group we only want to trigger unattended upgrades of security packages and not all other packages (as we do other host groups).

The reason of doing this is because of BBB's "sensitivity" with package upgrades. ;)

I hope that this is clear now?

Reply:
ok. makes sense